deploy/001_HypercertMinter_Upgrade.ts

```diff
@@ -0,0 +1,43 @@
+import { ethers, upgrades } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const deploy: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments } = hre; // we get the deployments and getNamedAccounts which are provided by hardhat-deploy.
+  const { save, get } = deployments; // The deployments field itself contains the deploy function.
+
+  try {
+    await get("HypercertMinterV0");
```

Comment:
I would stick these well-known strings in a constants file personally

Reply:
will do